src/js/dismiss.js

```diff
@@ -5,87 +5,69 @@
  */
 
 ( function() {
-
 	'use strict';
 
 	window.addEventListener( 'DOMContentLoaded', init, false );
 
 	function init() {
-
-		var els, i;
+		let els, i;
 
 		// Dismissible elements.
 		els = document.querySelectorAll( '.cl-countdown, .cl-notice' );
 
 		for ( i = 0; i < els.length; i++ ) {
-			setupDismiss( els[i] );
+			setupDismiss( els[ i ] );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['setupDismiss' was used before it was defined.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4811013)